ObjectMapper/Core/Mapper.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-public protocol Mappable {
+public protocol Mappable:Hashable {
```

Comment:
I believe that `Mappable` must be `Hashable` is a bad decision. Instead, You can add methods below in generic extensions of `Mapper`:

```swift
extension Mapper where N: Hasable {
    public func mapSet(JSONString: String) -> Set<N> { ... }
    ...
}
```